packages/okta-angular/test/e2e/harness/src/app/app.module.ts

```diff
@@ -31,15 +48,20 @@ const appRoutes: Routes = [
     canActivate: [ OktaAuthGuard ]
   },
   {
-    path: 'login',
-    component: OktaLoginRedirectComponent
+    path: 'protected-with-data',
+    component: ProtectedComponent,
+    canActivate: [ OktaAuthGuard ],
+    data: {
+      authGuardRedirect: onFailedAuthGuard
```

Comment:
idk if this will always be a redirect. We may consider renaming

Reply:
Updated to: `onAuthRequired`